src/arrays/ArrayUtil.java

```diff
@@ -11,85 +11,167 @@
      * Create an array with the given size, fill it with the different values and print it.
      */
     public static void createArray(int size) {
-
+        System.out.println("createArray");
+
+        int[] arr = new int[size];
+        for (int i : arr) {
+            System.out.print(i + ", ");
+        }
+        System.out.println();
+        System.out.println("_______________________");
     }
 
     /**
      * Create two dimensional array with the given sizes, fill it with different values and print it.
      */
     public static void create2dArray(int x, int y) {
+        System.out.println("create2dArray");
+
+        int[][] arr1 = new int[x][y];
+        for (int i = 0; i < y; i++) {
+            System.out.print(i + ", ");
+        }
+        System.out.println();
+        for (int j = 0; j < x; j++) {
+            System.out.print(j + ", ");
+        }
+        System.out.println();
+        System.out.println("_______________________");
     }
 
+
     /**
      * Add the given value at the beginning of the given array
      *
      * @return new array.
      */
     public static int[] addFirst(int[] array, int valueToAdd) {
+        System.out.println("addFirst");
 
-        return null;
+        int[] arr = new int[array.length];
+        arr[0] = valueToAdd;
+        System.arraycopy(arr, 0, array, 0, 1);
+        return array;
     }
 
+
     /**
      * Implement a method that will check if the given value contains in the array
-     *
-     * @return true if the array contains the given value, otherwise return false.
      */
-    public static boolean contains(int[] array, int value) {
-
+    public static boolean contains(int[] array1, int value) {
+        System.out.println("contains");
+
+        for (int i : array1) {
+            if (value == array1[i]) {
+                return true;
+            }
+        }
         return false;
     }
 
     /**
-     * @param array An array to search in.
+     * @param array2 An array to search in.
      * @param value The value to search for.
      * @return the first index of value in the array, if array does not contain given element return -1.
      */
-    public static int indexOf(int[] array, int value) {
-
-        return 0;
+    public static int indexOf(int[] array2, int value) {
+        System.out.println("indexOf");
+
+        for (int i : array2) {
+            if (value == array2[i]) {
+                return value;
+            }
+        }
+        return -1;
     }
 
     /**
      * Remove the element by the given index.
      * Print new array
      */
-    public static void remove(int[] array, int index) {
-
+    public static void remove(int[] array3, int index) {
+        System.out.println("remove");
+
+        int[] arr = new int[array3.length - 1];
+        for (int i : array3) {
+            System.out.print(i + ", ");
+        }
+        System.out.println();
+        if (index > array3.length - 1) {
```

Comment:
Also here you can check if **index < 0** and no need to continue invocation this method, just put **return;** after System.out.print("index chka");